mlir-tensorrt/compiler/lib/Dialect/TensorRTRuntime/Transforms/BufferizableOpInterfaceImpl.cpp

```diff
@@ -49,16 +56,16 @@ struct EnqueueOpInterface
   /// outputs in our use-case.
   bool bufferizesToMemoryRead(Operation *op, OpOperand &opOperand,
                               const bufferization::AnalysisState &state) const {
-    EnqueueOp callOp = cast<EnqueueOp>(op);
```

Comment:
NFC refactor changes should go in a separate PR.

Reply:
Sure. I will club all the NFC changes together in a separate PR. I am making changes in the same PR to track changes as of now.